src/sentry/runner/commands/run.py

```diff
@@ -335,6 +335,12 @@ def cron(**options):
     type=int,
     help="Time (in milliseconds) to wait before closing current batch and committing offsets.",
 )
+@click.option(
+    "--concurrency",
+    default=50,
```

Comment:
@nikhars I believe this is the value that we determined to be optimal after testing. Is that accurate?

Reply:
50 is too high. I think what we use in production is 10. What we should do is get this value from production and put that value as default.

At a higher concurrency level, the number of context switches would overwhelm the gain we get by having threads.

Reply:
It definitely seemed high to me. But it's the number that appeared to be set in prod looking at our metrics.

Reply:
Ok, the actual value we use is 5, I don't know why the metrics made it look like 50 to me. 5 seems like a more reasonable number to use across the board.